docker-compose.yml

```diff
@@ -111,8 +111,8 @@ services:
       DATABASE_HOST: storage-db
       DATABASE_PORT: 5432
       DATABASE_NAME: ods
-      DATABASE_USER: postgrest
-      DATABASE_PW: postgres-wrapper-pw
+      DATABASE_USER: ods_admin
```

Comment:
I think if you change the credentials here, you have to update the liquibase properties (`/storage/liquibase/changelog.xml`) accordingly.

Reply:
This is the old password (=equal to master branch credentials) .
I changed it because the liquibase service was failing (because of \r signs in the entrypoint.sh file) back then. 
I changed it back to the original values after the fix.